instance/static/html/instance/index.html

```diff
@@ -27,10 +27,12 @@
     <ul class="side-nav" ng-if="instanceList">
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
-        <div class="status-icon" ng-switch="instance.progress">
-          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="In progress" ng-switch-when="running"></i>
-          <i class="fa fa-times" tooltip-placement="top" tooltip="Failed" ng-switch-when="failed"></i>
-          <i class="fa fa-check" tooltip-placement="top" tooltip="Success" ng-switch-when="success"></i>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Waiting for server" ng-switch-when="waiting"></i>
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Configuring server" ng-switch-when="configuring"></i>
+          <i class="fa fa-times" tooltip-placement="top" tooltip="Error" ng-switch-when="error"></i>
+          <i class="fa fa-times" tooltip-placement="top" tooltip="Configuration Failed" ng-switch-when="failed"></i>
+          <i class="fa fa-check" tooltip-placement="top" tooltip="Running" ng-switch-when="running"></i>
```

Comment:
Could we instead do this generically and re-use the existing `status.description` attribute (which defaults to the docstring):

```
<div class="status-icon">
    <i ng-attr-class="{{ status.is_steady && (status.is_healthy && 'fa fa-check' || 'fa fa-times') || 'fa fa-ellipsis-h' }}"
        tooltip-placement="top"
        ng-attr-tooltip="{{status.description}}">
    </i>
</div>
```